### PR TITLE
[Docs] Build Chinese Docs

### PR DESCRIPTION
**Pull request by congee524**
you can preview the results at https://congee524-mmaction2.readthedocs.io/zh_CN/latest/

### CODE REVIEW

**Comment by dreamerlin** [comment]
image
We can use 'sed -i' to replace ’# xxxxx‘ with Chinese words

File: docs_zh_CN/merge_docs.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+sed -i 's/#/#&/' detection_models.md
+sed -i 's/md###t/html#t/g' localization_models.md
+sed -i 's/md###t/html#t/g' recognition_models.md
+sed -i 's/md###t/html#t/g' detection_models.md
+sed -i "s/md###t/html#t/g" demo.md
+
+sed -i 's/# Preparing/# /g' prepare_data.md
+sed -i 's/#/#&/' prepare_data.md
+
+sed -i '1i\# Action Localization Models' localization_models.md
```
Comment (innerlee) [inline docs_zh_CN/merge_docs.sh:39]: Translate L39-41

File: docs_zh_CN/stat.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    [f'   - {t}: {c}' for t, c in zip(papertypes, papercounts)])
+
+modelzoo = f"""
+# 总览
+
+* 论文数量: {len(alldatapapers)}
+{countstr}
+
+
+有关受支持的动作算法，可参见 [模型总览](modelzoo.md)。
```
Comment (innerlee) [inline docs_zh_CN/stat.py:159]: 动作 -> 视频理解

**Review by innerlee — APPROVED** [review]

**Comment by congee524** [comment]
waiting for CN Deeo merged #717

File: docs_zh_CN/index.rst [hunk 3]
```diff
@@ -58,6 +58,12 @@
 
    api.rst
 
+.. toctree::
+   :caption 中英文文档
```
Comment (innerlee) [inline docs_zh_CN/index.rst:62]: 语言切换

Comment (congee524) [reply docs_zh_CN/index.rst:62]: This attempt failed, unable to switch the language in the navigation area

**Review by hellock — APPROVED** [review]

**Comment by dreamerlin** [comment]
remind to rename `cn_docs` -> `docs_zh_CN`

**Pull request merged**